lib/report.js

```diff
@@ -150,6 +195,113 @@ class Report {
     return mergeProcessCovs(v8ProcessCovs)
   }
 
+  /**
+   * If --all is supplied we need to fetch a list of files that respects
+   * include/exclude that will be used to see the coverage report with
+   * empty results for unloaded files
+   * @returns {Promise.<Array.<string>>}
+   */
+  async getFileListForAll () {
```

Comment:
I don't think this method would need to be `async`, since we're using `globSync`; alternatively, perhaps we could expose `glob` in exclude.

Reply:
I think this is just an oversight on my part. Thanks for catching.